src/views/Login.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <v-card class="card">
+    <img class="logo" :src="require('@/assets/images/login-title.png')"/>
+    <div v-if="$apollo.loading" class="loading-wrapper">
+      <v-progress-circular
+                           indeterminate
+                           color="primary"
+                           :size="40" />
+      <h5>מנסה להתחבר</h5>
+    </div>
+    <v-form class="form" v-else>
+        <v-text-field
+        single-line
+        outlined
+        prepend-inner-icon="perm_identity"
+        placeholder="הזן שם משתמש"
+        v-model="userName"
+        />
+        <v-text-field
+        :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
+        @click:append="showPassword = !showPassword"
+        :type="showPassword ? 'text' : 'password'"
+        single-line
+        outlined
+        prepend-inner-icon="mdi-key-outline"
+        placeholder="הזן סיסמא"
+        v-model="password"
+        />
+        <v-checkbox
+        label="זכור אותי"
+        v-model="rememberLogin"/>
+        <h5 class='error-message'>{{this.errorMessage}}</h5>
+        <v-btn
+        class="login-button"
+        @click="login">
+            התחבר
+        </v-btn>
+    </v-form>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { getModule } from 'vuex-module-decorators';
+import { Component, Vue } from 'vue-property-decorator';
+
+import StoreModule from '@/store/storeModule';
+import { getLoggedInUser } from '@/db-service/Users/queries';
+import User from '@/models/User';
+
+const worngAuthMessage = 'אוף! שם המשתמש ו/או הסיסמא לא נכונים. פנה למנהל המערכת כדי לשחזר אותם';
+
+const dbErrorMessage = 'אופס! יש תקלה במערכת, נסה שוב מאוחר יותר';
+
+@Component
+export default class Login extends Vue {
+  private userName = '';
+
+  private password = '';
+
+  private rememberLogin = false;
+
+  private showPassword = false;
+
+  private errorMessage = '';
+
+  private storeModule = getModule(StoreModule, this.$store);
+
+  login() {
+    this.$apollo.query({
+      query: getLoggedInUser,
+      variables: {
+        userName: this.userName,
+        password: this.password,
+      },
+    }).then((data) => {
+      if (data.data.loggedInUser !== undefined) {
+        const loggedInUsers: User[] = data.data.loggedInUser.nodes;
+        if (loggedInUsers.length !== 0) {
+          this.storeModule.setUserId(loggedInUsers[0].id.toString());
+          if (this.rememberLogin) {
+            localStorage.setItem('userName', this.userName);
+            localStorage.setItem('password', this.password);
+          }
+          this.$router.replace('/');
+        } else {
+          this.errorMessage = worngAuthMessage;
+        }
+      }
+    })
+      .catch(() => { this.errorMessage = dbErrorMessage; });
+  }
+}
+</script>
+
+<style scoped>
+  .card {
```

Comment:
הכפתור של ההתחבר לא צריך להיות למטה? הכרטיס ארוך מדי לעומת התוכן שלו, אצלי הוא ממש נראה מתוח, אפשר להקטין את האורך ולמרכז את התוכן

Reply:
It looks that way because of there is an error message it takes more space on the card, so in order to prevent the size change when the error pops, I used a bigger card in advanced

Reply:
אבל לדעתי יש יותר מדי רווח גם כשיש את ההודעה אצלי יש מלא רווח למטה

Reply:
I will look at the bottom space, but the space abover the login button is gonna stay as it is so the card height wont change

Reply:
If you use "v-if" the element is not on the DOM page, so the space shouldn't be there. The "v-if" attribute makes elements to disappear from both UI and DOM frames.

Reply:
That the problem - since the element isn't there usually, when it is the card's size change, which seems a bit weirs, that why the space should be "allocated" before.

Reply:
so you can use 'display: hidden' conditionally with the title will stay in the dom but not displayed